request_handlers.cc

```diff
@@ -20,7 +27,9 @@ void EchoRequestHandler::handleRequest(const char* request_buffer,
   response_buffer_size = header_size + request_buffer_size;
 
   // TODO: need to check if we need to resize response buffer, if response is huge
```

Comment:
Do we still need to resize the buffer if we're creating it after we know the final size?